.github/workflows/build_package.yml

```
@@ -149,23 +163,39 @@ jobs:
       # is not enough to just invoke it directly because then pip managed
       # console scripts (like cmake, ninja) will not be on the path.
       ##########################################################################
-      - name: Main distribution (Linux)
-        if: "matrix.build-package == 'main-dist-linux'"
+      - name: Main distribution (Linux for x86_64)
+        if: >
+          "matrix.build-package == 'main-dist-linux' &&
+          matrix.build-family == 'linux-x86_64'"
         shell: bash
         run: |
           docker run --rm -w=/work \
             -v $PWD:/work \
             "${MANYLINUX_X86_64_IMAGE}" \
             bash -c 'export PATH=/opt/python/cp39-cp39/bin:$PATH; python ./c/build_tools/github_actions/build_dist.py main-dist'
 
+      - name: Main distribution (Linux for Arm64)
+        if: >
+          "matrix.build-package == 'main-dist-linux' &&
+          matrix.build-family == 'linux-aarch64'"
+        shell: bash
+        run: |
+          docker run --rm -w=/work \
+            -v $PWD:/work \
+            "${MANYLINUX_AARCH64_IMAGE}" \
+            bash -c 'export PATH=/opt/python/cp39-cp39/bin:$PATH; python ./c/build_tools/github_actions/build_dist.py main-dist'
+
       ##########################################################################
       # py-runtime-pkg
       # Builds the iree-runtime wheels.
       # One step per OS.
       ##########################################################################
 
-      - name: Build runtime wheels (Linux)
-        if: "matrix.build-package == 'py-runtime-pkg' && matrix.build-family == 'linux'"
+      - name: Build runtime wheels (Linux, x86_64 and Arm64)
+        if: >
+          "matrix.build-package == 'py-runtime-pkg' &&
+          (matrix.build-family == 'linux-x86_64' ||
+          matrix.build-family == 'linux-aarch64')"
```

How about shortening these checks to this?

* It's a bit over 80 characters, but we don't use line length limits in .yml files
* Less risk of a typo, and naturally extends to any future `linux-*` builds

```suggestion
        if: "matrix.build-package == 'py-runtime-pkg' && contains(matrix.build-family, 'linux')"
```

Then you could either match the 'contains' style for other platforms or just revert the other diffs

```diff
-        if: "matrix.build-package == 'py-runtime-pkg' && matrix.build-family == 'windows'"
+        if: "matrix.build-package == 'py-runtime-pkg' && contains(matrix.build-family, 'windows')"
```

---

Ok, will fix